README.md

```diff
@@ -6,7 +6,36 @@ Exercism exercises in Haskell
 
 ## Contributing Guide
 
-Please see the [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md)
+Haskell is a great language! (no, really!). Here in the Haskell track we do not
+just want to help you learn this great language, but we have the ambition to raise
+and share the passion for it. So we very much appreciate your help to make this happen!
+Any contribution that improves our track is welcome. How could such a contribution
+look like?
+Perhaps you have found a problem in one of the exercises. This can be a simple typo.
```

Comment:
What about making these things a bulleted list? Makes it easier to scan the list of options.